updating Node example app documentation

mansona opened the pull request:
I wasn't sure if you wanted to replace the current documentation for a Node example, I only did because it was marked as "outdated". 

Let me know if you want me to leave it in :+1:

lukemelia commented:
LGTM. @ghedamat?

ghedamat commented:
@mansona going trough the commits it seems [ember-lightning](https://github.com/philipheinser/ember-lightning/commits/master) has been updated to work with 0.5.X

might be fair to keep both the links

LGTM otherwise

thanks!

ghedamat commented on docs/v0.5.x/lightning-strategy-examples.md:
```diff
@@ -40,11 +40,13 @@ end
-## Example Node app (outdated)
+## Example Node app
 
-This app does the same as the *Sinatra app* above, it supports the same index_key query param. It should help you to get up and running in seconds and dont worry about server code. You also need to deploy your ember app with `ember-cli-deploy` and `ember-deploy-redis`.
```
ok this mention of `ember-deploy-redis` is wrong.

maybe keep your description, rename the section to `example node apps` and leave at the end the link to ember-lightning?

mansona replied:
So I've updated my side to have the correct reference to `ember-cli-deploy-lightning-pack` and I'll update the previous reference when I'm incorporating it back in :+1:

mansona commented:
@ghedamat awesome, I'll put it back in an differentiate them based on Node framework so :+1:

mansona commented:
@ghedamat I've added the previous docs back in and updated their reference to ember-cli-deploy-lightning-pack. So this should be good to go :tada: :smile:

ghedamat commented:
@mansona perfect!

Last request  could you squash into one commit?

mansona commented:
@ghedamat cool that's done now :+1: that's my first squash so it took a bit of googling :wink:

ghedamat commented:
@mansona thanks a lot!
